docs/docs/sourcing-from-gentics-mesh.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Sourcing from Gentics Mesh
+---
+
+## Using Gentics Mesh
+
+[Gentics Mesh](https://getmesh.io) is an open-source self-hosted headless CMS that supports GraphQL. Contents like posts, products, images and more are represented by elements called "nodes". These nodes can then be queried with the familiar GraphQL syntax.
+
+The model of nodes can be defined within "schemas". In our demo we have schemas for vehicles, images and categories.
+
+## Getting started
+
+In this guide you'll create a complete project capable of querying data from a Gentics Mesh server.
+
+### Start Mesh
+
+Gentics Mesh is a self-hosted CMS. You can run it for example via docker:
```

Comment:
```suggestion
Gentics Mesh is a self-hosted CMS. You can run it for example via [Docker](https://www.docker.com/):
```

Let's link to Docker in case people are not familiar with it